p2p/src/main/kotlin/maru/p2p/MaruPeerManager.kt

```diff
@@ -142,10 +142,10 @@ class MaruPeerManager(
         return
       }
       synchronized(connectionInProgress) {
-        if (p2pNetwork.peerCount >= maxPeers || connectionInProgress.contains(peer.nodeIdBytes)) {
+        if (p2pNetwork.peerCount >= maxPeers || connectionInProgress.contains(peer.nodeId)) {
```

Comment:
```suggestion
        if (getPeers().size >= maxPeers || connectionInProgress.contains(peer.nodeId)) {
```
Since `p2pNetwork` implements it via `MaruPeerManager`, using `p2pNetwork` here doesn't seem right

Reply:
fixed